misc.h

```diff
@@ -37,19 +37,114 @@
 #endif
 
 #define IS_MR_CHANNEL(x)       ((x) <= MR_CHANNEL_LAST)
-#define IS_FREQ_CHANNEL(x)     ((x) >= FREQ_CHANNEL_FIRST && (x) <= FREQ_CHANNEL_LAST)
```

Comment:
There are 2 (emulated) VFOs (variable frequency oscillator) in this radio.

There is a memory mode, and frequency mode, so MR_CHANNEL or FREQ_CHANNEL

Reply:
Got it!
Will revert to FREQ in the next commit. Just need to finish the chirp driver to be sure everything is in sync and working as expected.